### PR TITLE
Conversational intelligence challenge solution for team kAIb

### PR DESCRIPTION
**Pull request by gmkim90**
Conversational intelligence challenge solution for team kAIb.

If you have any questions, please contact authors directly.

### CODE REVIEW

File: 2017/solutions/kaib/DA/requirements.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+tensorflow_gpu==0.13.0
```
Comment (madrugado) [inline 2017/solutions/kaib/DA/requirements.txt]: it seems to me, there are some dependencies missing, like numpy and nltk

**Comment by gmkim90** [comment]
Sorry for late reply.
Could you follow steps in readme?
https://github.com/gmkim90/convai/blob/master/2017/solutions/kaib/readme.txt

Which parts do you have trouble with?

**Comment by madrugado** [comment]
Ken, please read my comment above.

File: 2017/solutions/kaib/DA/requirements.txt (hunk 1 above)
Comment (gmkim90) [reply 2017/solutions/kaib/DA/requirements.txt]: What makes you think that some dependencies are missing?

Comment (madrugado) [reply 2017/solutions/kaib/DA/requirements.txt]: because you're using additional software like NLTK, so you should add it as perendency explicitly

Comment (gmkim90) [reply 2017/solutions/kaib/DA/requirements.txt]: We updated readme & requirements.txt, which lists all required external libraries.

**Pull request merged**